src/Modules/Backend/assets/Backend/webpack/scss/components/_date-time-picker.scss

```diff
@@ -45,9 +46,9 @@ span.flatpickr-day.nextMonthDay:nth-child(n+8) {
 }
 
 .flatpickr-calendar.arrowTop {
-
-  &:after,
-  &:before {
+  &::after,
+  &::before {
     border-bottom-color: $primary;
   }
-}
\ No newline at end of file
+}
+/* stylelint-enable */
```

Comment:
why not add an empty line at the end?

Reply:
fixed 4599cb3c456f80fc72cb633661033a3486890dd3